events/diskpersist.go

```diff
@@ -145,7 +148,7 @@ func (dp *DiskPersistence) resumeLog() error {
 		return dp.initLogFile()
 	}
 
-	fi, err := os.Open(filepath.Join(dp.primaryDir, lfr.Path))
+	fi, err := os.OpenFile(filepath.Join(dp.primaryDir, lfr.Path), os.O_RDWR, 0)
```

Comment:
LGTM but I'd like a comment above here for the `0` filemode, OpenFile doesn't use the file mode if `O_CREATE` isn't passed but it's kinda scary to see.